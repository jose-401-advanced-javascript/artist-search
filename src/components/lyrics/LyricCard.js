import React from 'react';
import PropTypes from 'prop-types';

const LyricCard = ({ lyric }) => {

  return (
    <div>
      <pre>{lyric}</pre>
    </div>
  );
};

LyricCard.propTypes = {
  lyric: PropTypes.string.isRequired
};

export default LyricCard;