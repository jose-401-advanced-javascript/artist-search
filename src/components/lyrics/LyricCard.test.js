import React from 'react';
import { shallow } from 'enzyme';
import LyricCard from './LyricCard';

describe('LyricCard component', () => {
  it('renders LyricCard', () => {
    const wrapper = shallow(<LyricCard lyric={'asdga'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
