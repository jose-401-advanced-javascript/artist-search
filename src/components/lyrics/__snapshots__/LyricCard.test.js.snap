// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LyricCard component renders LyricCard 1`] = `
<div>
  <pre>
    asdga
  </pre>
</div>
`;
